server/ntb/publish/ntb_nitf.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.metadata.item import ITEM_TYPE, CONTENT_TYPE
+from superdesk.publish.formatters.nitf_formatter import NITFFormatter
+from xml.etree.ElementTree import SubElement
+import logging
+import re
+logger = logging.getLogger(__name__)
+
+EMBED_RE = re.compile(r"<!-- EMBED START (([a-zA-Z]+) {id: \"(?P<id>.+?)\"}) -->(.*)"
+                      r"<!-- EMBED END \1 -->", re.DOTALL)
+
+
+class NTBNITFFormatter(NITFFormatter):
+
+    def can_format(self, format_type, article):
+        """
+        Method check if the article can be formatted to NTB NIT
+        :param str format_type:
+        :param dict article:
+        :return: True if article can formatted else False
+        """
+        return format_type == 'ntbnitf' and article[ITEM_TYPE] == CONTENT_TYPE.TEXT
+
+    def _format_tobject(self, article, head):
+        category = ''
+        for s in article.get('subject', []):
+            if s.get('scheme') == 'category':
```

Comment:
Category is a custom field added for NTB. Can we avoid this check and make it more generic?
For example we can test is scheme is not subject to get qcode.